DB audit logging web page adjustment, synchronous modification operat… (#19448)
pingcap/docs: tidb-cloud/tidb-cloud-auditing.md

- You are using a TiDB Cloud Dedicated cluster. Audit logging is not available for TiDB Cloud Serverless clusters.
- You are in the `Organization Owner` or `Project Owner` role of your organization. Otherwise, you cannot see the database audit-related options in the TiDB Cloud console. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).

## Enable audit logging

TiDB Cloud supports recording the audit logs of the Dedicated cluster to your cloud storage service. Before enabling Database audit logging, you need to prepare your cloud storage service at the cloud service provider where the cluster is located. To enable it, take the following steps:

> 
> 1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
> 
>     > **Tip:**
>     >
>     > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
> 
> 2. Click the name of your target cluster to go to its overview page, and then click **DB audit logging** in the left navigation pane.
> 
> 3. Database audit logging is related to cloud service providers. Therefore, the operation guide for enabling the audit logging is different for clusters deployed on different cloud service providers.

Reviewer: ```suggestion
```
Move these steps to "Step 2. Configure Amazon S3 access" and "Step 2. Configure GCS access" sections to consolidate TiDB Cloud console operations and streamline Account ID retrieval process.

#### Step 2. Configure Amazon S3 access

> 1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.

Reviewer: ```suggestion
1. Get the TiDB Cloud Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

        > **Tip:**
        >
        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.

    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.

>     4. In the **Enable Database Audit Logging** dialog, locate the **AWS IAM Policy Settings** section, and record **TiDB Cloud Account ID** and **TiDB Cloud External ID** for later use.

Reviewer 2: Here we use **TiDB Cloud Account ID**, but in line 43 you changed "TiDB Cloud account ID" to "Account ID". I think users will get confused by these two terms.

Reviewer: Fix in https://github.com/pingcap/docs/pull/19448/commits/3773172f7ec07ed132e639fde084c608ca44d8b4
```diff
- 1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
+ 1. Get the TiDB Cloud Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

#### Step 2. Configure Amazon S3 access

1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.

> 

Reviewer: ```suggestion
```

> 1. Get the TiDB Cloud account ID and the External ID of the TiDB cluster that you want to enable audit logging.

Reviewer: ```suggestion
1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

>     1. On the DB Audit Logging page, click the **Enable** button to enter the audit configuration page.
>     2. On the right side of the configuration page, you can get the **AWS IAM Popicy Settings**. Please record the TiDB Cloud Account ID and the External ID for later use.

Reviewer: ```suggestion

    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

        > **Tip:**
        >
        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.

    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.
    4. In the **Enable Database Audit Logging** dialog, locate the **AWS IAM Policy Settings** section, and record **TiDB Cloud Account ID** and **TiDB Cloud External ID** for later use.
```

#### Step 3. Enable audit logging

In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:

1. In the **Bucket URI** field, enter the URI of your S3 bucket where the audit log files are to be written.
2. In the **Bucket Region** drop-down list, select the AWS region where the bucket locates.
3. In the **Role ARN** field, fill in the Role ARN value that you copied in [Step 2. Configure Amazon S3 access](#step-2-configure-amazon-s3-access).
4. Click **Test Connection** to verify whether TiDB Cloud can access and write to the bucket.

    If it is successful, **The connection is successfully** is displayed. Otherwise, check your access configuration.

5. Click **Enable** to enable audit logging for the cluster.

    TiDB Cloud is ready to write audit logs for the specified cluster to your Amazon S3 bucket.

> **Note:**
>

> > - After enabling audit logging, if you make any new changes to the bucket URL, location, or ARN, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.

Reviewer: ```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URI, location, or ARN, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```

#### Step 3. Enable audit logging

In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:

> 1. In the **Bucket URI** field, enter the URI of your S3 bucket where the audit log files are to be written.

Reviewer: ```suggestion
1. In the **Bucket URL** field, enter the URI of your S3 bucket where the audit log files are to be written.
```

<img width="363" alt="image" src="https://github.com/user-attachments/assets/f7de9073-17c7-4e00-b3c1-1ba28d2d1fef">

Author: This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.

1. Get the Google Cloud Service Account ID of the TiDB cluster that you want to enable audit logging.

>     1. On the DB Audit Logging page, click the **Enable** button to enter the audit configuration page.
>     2. On the right side of the configuration page, you can get the **Google Cloud Server Account ID**. Please record the TiDB Cloud Account ID for later use.

Reviewer: ```suggestion
    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

        > **Tip:**
        >
        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.

    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.
    4. In the **Enable Database Audit Logging** dialog, locate the **Google Cloud Server Account ID** section, and record **Service Account ID** for later use.
```

#### Step 3. Enable audit logging

In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:

> 1. In the **Bucket URI** field, enter your full GCS bucket name.

Reviewer: ```suggestion
1. In the **Bucket URL** field, enter your full GCS bucket name.
```

Author: This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.

2. In the **Bucket Region** field, select the GCS region where the bucket locates.
3. Click **Test Connection** to verify whether TiDB Cloud can access and write to the bucket.

    If it is successful, **The connection is successfully** is displayed. Otherwise, check your access configuration.

4. Click **Enable** to enable audit logging for the cluster.

    TiDB Cloud is ready to write audit logs for the specified cluster to your GCS bucket.

> **Note:**
>

> > - After enabling audit logging, if you make any new changes to bucket URI or location, you must click **Test Connection** to re-verify TiDB Cloud's access configuration to the bucket, and then click **Enable** for the changes to take effect.
> > - To remove TiDB Cloud's access to your GCS bucket, simply delete the trust policy granted to this cluster in your GCS bucket.

Reviewer: ```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URL or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
> - To remove TiDB Cloud's access to your GCS bucket, delete the trust policy granted to this cluster in the Google Cloud console.
```

#### Step 3. Enable audit logging

In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:

1. In the **Bucket URI** field, enter your full GCS bucket name.
2. In the **Bucket Region** field, select the GCS region where the bucket locates.
3. Click **Test Connection** to verify whether TiDB Cloud can access and write to the bucket.

    If it is successful, **The connection is successfully** is displayed. Otherwise, check your access configuration.

4. Click **Enable** to enable audit logging for the cluster.

    TiDB Cloud is ready to write audit logs for the specified cluster to your GCS bucket.

> **Note:**
>

> > - After enabling audit logging, if you make any new changes to the bucket URL or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.

Reviewer: ```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URI or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```
